pkg/discovery/dns/provider.go

```diff
@@ -107,8 +108,10 @@ func GetQTypeName(addr string) (qtype string, name string) {
 // Resolve stores a list of provided addresses or their DNS records if requested.
 // Addresses prefixed with `dns+` or `dnssrv+` will be resolved through respective DNS lookup (A/AAAA or SRV).
 // defaultPort is used for non-SRV records when a port is not supplied.
-func (p *Provider) Resolve(ctx context.Context, addrs []string) {
+func (p *Provider) Resolve(ctx context.Context, addrs []string) tsdberrors.MultiError {
 	resolvedAddrs := map[string][]string{}
+	errorList := tsdberrors.MultiError{}
```

Comment:
We use to use just `errs` name for those

Reply:
Sure.